content/espanol/python-guess-number/answer-key.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Python Adivina el Número - Clave de Respuestas"
```

Comment:
"Python: Adivina el Número - Respuestas"

Reply:
resolved and updated